src/antelope/config/AntelopeDebug.ts

```diff
@@ -0,0 +1,195 @@
+
+
+// creamos una clase llamada AntelopeDebug que nos permita almacenar el estado de estar
```

Comment:
would you mind translating this and the comment at line 11 into english?

Reply:
I used that text to generate code. Is not useful anymore. 
Thanks!